internal/cli/cmd/cluster/describe.go

```diff
@@ -225,44 +225,42 @@ func showEndpoints(c *appsv1alpha1.Cluster, svcList *corev1.ServiceList, out io.
 	tbl.Print()
 }
 
-func showDataProtection(backupPolicies []dpv1alpha1.BackupPolicy, backups []dpv1alpha1.Backup, out io.Writer) {
-	if len(backupPolicies) == 0 {
+func showDataProtection(backupPolicies []dpv1alpha1.BackupPolicy, backupSchedules []dpv1alpha1.BackupSchedule, out io.Writer) {
+	if len(backupPolicies) == 0 || len(backupSchedules) == 0 {
 		return
 	}
-	tbl := newTbl(out, "\nData Protection:", "AUTO-BACKUP", "BACKUP-SCHEDULE", "TYPE", "BACKUP-TTL", "LAST-SCHEDULE", "RECOVERABLE-TIME")
+	tbl := newTbl(out, "\nData Protection:", "BACKUP-REPO", "AUTO-BACKUP", "BACKUP-SCHEDULE", "BACKUP-METHOD", "BACKUP-RETENTION")
 	for _, policy := range backupPolicies {
```

Comment:
It is recommended to iterate over the backupSchedule in the outer loop, and iterate over the corresponding policy in the inner loop.

```
if policy.Annotations[dptypes.DefaultBackupPolicyAnnotationKey] != "true" {
   continue
}
```

If policy is not default, the schedule will be ignored, that is not be expected, the schedule can specify one backup policy.